react/features/always-on-top/AlwaysOnTop.js

```diff
@@ -80,8 +90,8 @@ export default class AlwaysOnTop extends Component<*, State> {
      */
     _displayNameChangedListener({ formattedDisplayName, id }) {
         if (api._getOnStageParticipant() === id
-                && formattedDisplayName !== this.state.displayName) {
-            this.setState({ displayName: formattedDisplayName });
+                && formattedDisplayName !== this.state.formattedDisplayName) {
```

Comment:
You'll want to updated display name here too.

Reply:
thanks!

Reply:
Still needs to be updated here, right?

Reply:
That got removed on one of the rebases, thanks